src/server/v2.0/handler/model/retention.go

```diff
@@ -0,0 +1,74 @@
+package model
+
+import (
+	"encoding/json"
+	"github.com/goharbor/harbor/src/lib"
+	"github.com/goharbor/harbor/src/pkg/retention"
+	"github.com/goharbor/harbor/src/pkg/retention/policy"
+	"github.com/goharbor/harbor/src/server/v2.0/models"
+)
+
+// RetentionPolicy ...
+type RetentionPolicy struct {
+	*policy.Metadata
+}
+
+// ToSwagger ...
+func (s *RetentionPolicy) ToSwagger() *models.RetentionPolicy {
+	var result models.RetentionPolicy
+	lib.JSONCopy(&result, s)
+	return &result
+}
+
+// FromSwagger ...
+func FromSwagger(policy *models.RetentionPolicy) *RetentionPolicy {
```

Comment:
Rename it to `NewRetentionPolicyFromSwagger`. FromSwagger is too general for the package `model`.